client/galaxy/style/scss/theme/blue.scss

```diff
@@ -145,7 +145,7 @@ $form-border: darken($form-heading-bg, 20%);
 // Table
 $table-heading-bg: $panel-bg-color;
 $table-bg-accent: darken($panel-bg-color, 5%);
-$table-border-color: $border-color;
+$table-border-color: rgba(0, 0, 0, 0);
```

Comment:
See comment above. Border colors should be either transparent or derived from current border-colors which are derived from text-colors.

Reply:
Thank you, I will change this.